src/oc/storage/api/entries.clj

```diff
@@ -267,15 +285,73 @@
   ;; Responses
   :handle-ok (fn [ctx] (entry-rep/render-entry-list org-slug board-slug
                           (:existing-entries ctx) (:access-level ctx) (-> ctx :user :user-id)))
-  :handle-created (fn [ctx] (let [new-entry (:created-entry ctx)]
+  :handle-created (fn [ctx] (let [new-entry (:created-entry ctx)
+                                  existing-board (:existing-board ctx)]
                               (api-common/location-response
                                 (entry-rep/url org-slug board-slug (:uuid new-entry))
-                                (entry-rep/render-entry org-slug board-slug new-entry [] []
+                                (entry-rep/render-entry org-slug existing-board new-entry [] []
                                   :author (-> ctx :user :user-id))
                                 mt/entry-media-type)))
   :handle-unprocessable-entity (fn [ctx]
     (api-common/unprocessable-entity-response (:reason ctx))))
 
+;; A resource for operations to publish a particular entry
+(defresource publish [conn org-slug board-slug entry-uuid]
+  (api-common/open-company-authenticated-resource config/passphrase) ; verify validity and presence of required JWToken
+
+  :allowed-methods [:options :post]
+
+  ;; Authorization
+  :allowed? (by-method {
+    :options true
+    :post (fn [ctx] (access/allow-authors conn org-slug (:user ctx)))})
+
+  ;; Media type client accepts
+  :available-media-types (by-method {
+                            :post [mt/entry-media-type]})
+  :handle-not-acceptable (by-method {
+                            :post (api-common/only-accept 406 mt/entry-media-type)})
+
+  ;; Possibly no data to handle
+  :malformed? (by-method {
+    :options false
+    :post (fn [ctx] (api-common/malformed-json? ctx true))}) ; allow nil
+  :processable? (by-method {
+    :options true
+    :post (fn [ctx] (let [entry-props (:data ctx)]
+                      (or (nil? entry-props) ; no share is fine
```

Comment:
probably this comment came from the old API where you could share a story while publishing it.

Reply:
agree on the comment, it's updated